ProjectEureka/project-eureka-flutter
# PE-65 | Create Rating Service

What I Did: 

- Created a rating service file that will send the rating and current user_id via HTTP PUT to the backend. 

How to Test: 

- Sign up if you don't already have an account. 
- Post a new question and give a rating after the question has been submitted. 
- Check MongoDB and see if the rating you gave was added to your account. 
- Post a few more questions and give a rating. Check MongoDB again if the ratings update as well as the average rating.
<img width="116" alt="rating" src="https://user-images.githubusercontent.com/60668425/112912558-a0b62b80-90ac-11eb-8a89-566650cd4e0e.PNG">

Note: Rating does not take in 0.5 values for now until the backend is updated to take in a double instead of an int.
